clouway-pcache-client-redis/src/main/java/com/clouway/api/pcache/extensions/redis/RedisCacheManagerFactory.java

```diff
@@ -58,15 +59,14 @@ public static CacheManager create(String redisHost) {
    * @return the newly created cache manager
    */
   public static CacheManager create(String redisHost, NamespaceProvider namespaceProvider) {
-    Jedis jedis;
-
+    JedisPool pool;
```

Comment:
I think it will be better if there is a possibility to pass JedisPool config, in case someone would like to use custom set up